src/main/java/AddCommand.java

```diff
@@ -1,15 +1,20 @@
 import java.io.IOException;
 
+/**
+   * AddCommand inherits Command
+   * @param command the main action of the command (todo,deadline,event)
+   * @param task the description of the task
+   * @param date the due date specified by user
+   */
+
 public class AddCommand extends Command {
 
     public AddCommand(String command, String task, String date) {
         super(command, task, date);
-        // TODO Auto-generated constructor stub
     }
 
     @Override
     void execute(TaskList taskList, Ui ui, Storage storage) throws IOException {
```

Comment:
A clear and concise name for the method! Perhaps you could include a JavaDoc header for this method as well?